### ISSUE TITLE
Is it possible to have a 3d image means the building in the image should look like actual buildings as in the 3d models 

### ISSUE THREAD

**Issue by yashj119** [w3reality/three-geo#23]

**Comment by Cobertos** [comment]
I think that's currently a WIP, see https://github.com/w3reality/three-geo/blob/master/src/models/building.js

**Comment by yashj119** [comment]
Ok means you are working on it? How soon you are planning for a release.

**Comment by Cobertos** [comment]
Nope, sorry, I've just been going through the repo and found the file with the comment in it.

**Comment by yashj119** [comment]
So you mean is it build already ?

**Comment by yashj119** [comment]
Do you have any working example? Or how i can use this feature with
three-geo.

**Comment by Cobertos** [comment]
Not quite? If you look at the Twitter link in the link I pasted above...

https://twitter.com/w3reality/status/1063321659342434310

...whoever runs the account has an example that seems to work quite well, but there's no code on GitHub yet.

Probably need to wait on a response from @j-devel , but I'm interested as well.

**Comment by yashj119** [comment]
Ok so are you a developer or please let me know what you do ?

**Comment by Cobertos** [comment]
I'm just a user of the library, not a contributor (yet), and thought pointing out that `building.js` exists and is WIP but also a pretty far along WIP from the Twitter video might help.

**Comment by j-devel** [comment]
The Twitter video is a demo of a 3D building viz library (say, 'three-buidlings') that I developed a while ago and has been left unpublished (i.e. not oss) ever since.

Now I intend to merge it into this three-geo repo, and 'src/models/building.js' will be the destination.

I want to make time for this task, but I am afraid I can't say exactly when, though.

**Comment by yashj119** [comment]
Your work and ideas are really valuable and i think many users are waiting
to see this.

Thanks

**Comment by yashj119** [comment]
Hello,

Have you ever work with moment js. I need one help from you, i am using
this code moment().tz("America/Newyork") but the _d in moment object is not
updating please can you let me know why? It is always pointing to my local
time

**Comment by yashj119** [comment]
Hi j-devel,

Any update so far on the building.js. I think users need your three-geo and
building.js both. Please let me know how soon you are going to release.

Thanks
